### PR TITLE
feat(logs): enable logs in console to display in argo

### PR DESCRIPTION
**Pull request by Passerino**
logs are now being sent to file and console on every env

### CODE REVIEW

File: src/modules/logger/logger.service.ts [hunk 1]
```diff
@@ -54,14 +35,31 @@ export class Logger extends NestLogger implements LoggerService {
...
+      maxFiles: '14d',
+    });
+
+    const console = new transports.Console({ format: logFormat });
 
     this.logger = createLogger({
       format: logFormat,
       level: 'debug',
-      transports: [transport],
+      transports: [console, file],
```
Comment (jrhender) [inline src/modules/logger/logger.service.ts:62]: @Passerino this means that we are logging to both `console` and `file` then?

Comment (Passerino) [reply src/modules/logger/logger.service.ts:62]: yes, I don't think we will look at the file log anytime soon, but wanted to keep backward compatibility.

Comment (jrhender) [reply src/modules/logger/logger.service.ts:62]: @hejkerooo do you see any issue with logging to a file in a kubernetes pod?

**Review by jrhender — APPROVED** [review]
Let's try it out 😄  If there are issues, we can revert

File: src/modules/logger/logger.service.ts (hunk 1 above)
Comment (Passerino) [reply src/modules/logger/logger.service.ts:62]: it's a rolling file so I don't foreseen any storage limit issues.

**Pull request merged**